src/arrayMethodSort.js

```diff
@@ -2,11 +2,24 @@
 
 /**
  * Implement method Sort
- */
+//  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) > String(b)) {
```

Comment:
move your compare function to a separate method to make the code clearer.

Reply:
and let's make this function return 0, 1, or -1 as the original sort function

Reply:
Frankly saying, I can't understand how to solve it...